### PR TITLE
Update Makefile & rsrc->rdef

### PR DESCRIPTION
**Pull request by BachToTheFuture**
Noticed that there was a Makefile, but thought it would be nice to convert it to a nicer Haiku Makefile format.
Also converted rsrc to rdef :)

### CODE REVIEW

File: src/Space.rdef [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+resource app_version {
+	major  = 1,
+	middle = 1,
+	minor  = 0,
+
+	variety = B_APPV_DEVELOPMENT,
+	internal = 0,
+
+	short_info = "Space Screen Saver",
```
Comment (humdingerb) [inline src/Space.rdef]: "Space screensaver" looks better.

File: src/Space.rdef [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+	$"0000000000000000000000003F3F3F00"
+	$"00000000003F000000000000193F1900"
+	$"00000000000000000000000000000000"
+};
+
+resource app_signature "application/x-vnd.SpaceScreenSaver";
+
+resource app_version {
+	major  = 1,
+	middle = 1,
```
Comment (humdingerb) [inline src/Space.rdef]: The ReadMe and elsewhere refers to the version as 1.2.

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: src/Space.rdef [hunk 3]
```diff
@@ -56,14 +56,14 @@ resource mini_icon array {
 resource app_signature "application/x-vnd.SpaceScreenSaver";
 
 resource app_version {
-	major  = 1,
-	middle = 1,
+	major  = 1.2,
+	middle = 1.2,
```
Comment (humdingerb) [inline src/Space.rdef]: Almost. :)
```
major = 1,
middle = 2,
minor = 0,
```

**Review by humdingerb — CHANGES_REQUESTED** [review]

File: src/Space.rdef (hunk 3 above)
Comment (BachToTheFuture) [reply src/Space.rdef]: Ah. I see :)

**Review by humdingerb — APPROVED** [review]

**Comment by humdingerb** [comment]
Thanks!

**Pull request merged**